relational.tex

```diff
@@ -670,10 +682,10 @@ \subsection{SQL}
 \subsection{IVM query plans and optimality}
 
 A relational algebra query can be implemented by multiple plans, each
```

Comment:
I cannot comment on the SQL section above. Can we refer to a section in the full paper for group by and aggregation?